lib/Ama.pm

```diff
@@ -96,6 +102,7 @@ $self->plugin("OAuth2Accounts" => {
   $r->delete('/questions/:question_id')->to('questions#remove')->name('remove_question'); # Delete from DB and redirect to questions
   $r->delete('/removeAll')->to('questions#removeAll')->name('removeAll'); #Delete every question
```

Comment:
@s1037989 I don't think I implemented this removeAll feature... I don't remember for sure though. @meyerben Check this out